terraform/messaging.tf

```diff
@@ -144,3 +161,37 @@ resource "aws_sqs_queue_policy" "ingest" {
     }]
   })
 }
+
+resource "aws_sqs_queue" "product_create" {
+  name = "${local.resource_prefix}-product-create-queue"
+  kms_master_key_id = aws_kms_key.product_create.id
+  message_retention_seconds = 7 * 24 * 60 * 60  # 1 week
+  visibility_timeout_seconds = 12 * 60 * 60     # 12 hours
+}
+
+resource "aws_sqs_queue" "async_update" {
+  name = "${local.resource_prefix}-async-update-queue"
+  kms_master_key_id = aws_kms_key.async_update.id
+  message_retention_seconds = 7 * 24 * 60 * 60  # 1 week
+  visibility_timeout_seconds = 12 * 60 * 60     # 12 hours
```

Comment:
Same - 12 hours seems a bit long for the default visibility timeout shouldn't the message get processed within minutes?